packit_service/worker/events/copr.py

```diff
@@ -67,6 +68,11 @@ def __init__(
         self.pkg = pkg
         self.timestamp = timestamp
 
+    @classmethod
+    def event_type(cls) -> str:
+        assert os.environ.get("PYTEST_VERSION"), "Should be initialized only during tests"
```

Comment:
Unfortunately in Visual Studio Code this trick does not work, it would be hard to use concrete events instead of abstract ones in tests?

Reply:
Just out of interest, what version of pytest are you running? I did unpin the version on Friday :eyes:

Reply:
Nevermind, I thought that using pytest directly I had the `PYTEST_VERSION` env variable already initialized (and not when calling tests from VS code) but I see now that it isn't initialized even when calling pytest from command line. So now we should always define this variable first? right?

Reply:
It's defined since pytest 8.2.0; we had pinned an older version till Friday when I unpinned and switched to the pytest from the PyPI… I really hate the fact we have different environments (for packit we run TF **and** Zuul), and `Makefile` also allows running tests from the container (they can easily diverge in the versions… I switched the pytest to PyPI exactly because of the different version in Fedora and EPEL).